### ISSUE TITLE
 More information about LCNet

### ISSUE THREAD

**Issue by a1wright** [fabvio/Cascade-LD#16]
Friend please help me.  Please explain to me how  the descriptors of different lanes could be classified simultaneously. I really don't understand how this happens. How does the output get 4 channels?

**Comment by fabvio** [comment]
Hi. It's a simple tensorial operation, I'm processing 4 different descriptors by using batched operations in pytorch.

**Comment by a1wright** [comment]
Thank you very much. I have one more question. Can you please provide graphs or a history of changes in the loss function? I would like to see how the gradient changes in ERFNet. You would help me a lot.

**Comment by fabvio** [comment]
I'm sorry, I won't have time for that. Also, it's not very clear what you aim to visualize, so I believe it would be best if you do it yourself.
